app/models/gobierto_people/person.rb

```diff
@@ -64,6 +65,24 @@ def owned_attending_events
       attending_events.person_events.where(collection_items: { container_id: id })
     end
 
+    def historical_charge(date = nil)
+      if date.present?
+        historical_charges.on_date(date).take
+      else
+        historical_charge(Date.current) || historical_charges.reverse_sorted.first
+      end
+    end
+
+    def charge(date = nil)
+      if historical_charges.exists?
+        historical_charge(date)&.name
+      else
+        return if charge_translations.blank?
+
+        charge_translations.fetch(I18n.locale, charge_translations&.values&.first)
```

Comment:
If `charge_translations` is a regular translate attribute why not replace the whole `else` block by a plain `super`? Would it work?

Reply:
Actually `charge` is overrided but it's not given by a superclass, is introduced with `translates` call in the model, calling super in this context generates an error saying that the method is not defined in any superclass. But if we create a superclass with the `translate  :charge` statement and the method is inherited from it, the `super` call raises an exception because the charges superclass doesn't accept arguments and this accepts an optional argument.